scripts/appleseed.package.py

```diff
@@ -211,10 +211,11 @@ def print_summary(self):
         print("  Configuration:             {0}".format(self.configuration))
         print("  Path to appleseed:         {0}".format(self.appleseed_path))
         print("  Path to appleseed headers: {0}".format(self.appleseed_headers_path))
+        print("  Path to Python 2.7:        {0}".format(self.python_path))
         print("  Path to Qt runtime:        {0}".format(self.qt_runtime_path))
         if os.name == "nt":
             print("  Path to platform runtime:  {0}".format(self.platform_runtime_path))
-        print("  Path to Python 2.7:        {0}".format(self.python_path))
+            print("  Path to OpenSSL DLLs:      {0}".format(self.openssl_path))
```

Comment:
> OpenSSL is required to send encrypted data to the backend right?

More precisely, it's to allow reaching the backend API via HTTPS.

> What about linux? Do we need to do some changes so that bench work on it?

appleseed.bench isn't supported on Linux yet. The effort to make it work on Linux should be fairly minor, it should be mostly about implementing CPU detection.